code/game/dna/dna_modifier.dm

```diff
@@ -253,9 +253,14 @@
 	return ..()
 
 /obj/machinery/dna_scannernew/crowbar_act(mob/user, obj/item/I)
-	if(default_deconstruction_crowbar(user, I))
-		for(var/obj/thing in contents) // in case there is something in the scanner
-			thing.forceMove(loc)
+	if(!panel_open)
+		return
+	. = TRUE
+	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+		return
+	for(var/obj/thing in contents) // in case there is something in the scanner
+		thing.forceMove(loc)
+	default_deconstruction_crowbar(user, I)	
```

Comment:
This can return `FALSE` if the machine has `NODECONSTRUCT` in it's `flags` var. Be sure to catch that before moving the content out of it.

Reply:
Addressed